lib/create-disputed-vote.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs')
```

Comment:
Maybe it's worth putting these under a `buidler/` folder to make it more clear it's used internally as a task

Reply:
The good thing is that this is a plain script that could be run with truffle exec as well :)
The buidler part of it is actually placed in the buidler config file